awx/settings/defaults.py

```diff
@@ -370,12 +372,7 @@
 }
 
 AUTHENTICATION_BACKENDS = (
-    'awx.sso.backends.LDAPBackend',
-    'awx.sso.backends.LDAPBackend1',
-    'awx.sso.backends.LDAPBackend2',
-    'awx.sso.backends.LDAPBackend3',
-    'awx.sso.backends.LDAPBackend4',
-    'awx.sso.backends.LDAPBackend5',
+    "ansible_base.authentication.backend.AnsibleBaseAuth",
```

Comment:
Are we actually ready to make this change? Does this "just work", or will it require some additional changes (especially on the UI)?